ipa-core/src/cli/playbook/ipa.rs

```diff
@@ -5,6 +5,7 @@ use std::{
     time::{Duration, Instant},
 };
 
+use bitvec::macros::internal::funty::Fundamental;
```

Comment:
oh this does not seem right - we shouldn't require this import

Reply:
hmm yeah, not sure how that got added.  I'll try to remove it.